server.c

```diff
@@ -123,9 +123,22 @@ int main(void)
             if (status == 0) printf("Other connection closed");
             if (status == -1) perror("recv");
             printf("%s", str);
-            if (send(new_fd, "Hello, world!", 13, 0) == -1)
+
+	    // send file named index.html whenever we get a request
+	    FILE* home_pg = fopen("index.html", "r");
+	    fseek(home_pg, 0L, SEEK_END);
+	    int size = ftell(home_pg);
+	    char* buffer = (char*)calloc(size, sizeof(char));
+	    if (buffer == NULL)
+	    	return 1;
```

Comment:
perror (to later be replaced by actual logging) here saying where we went wrong for easier/faster debugging